kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/source/S3SourceTaskTest.scala

```diff
@@ -100,7 +100,61 @@ class S3SourceTaskTest
           val props = DefaultProps
             .combine(
               Map(
-                "connect.s3.kcql" -> s"insert into ${bucketSetup.TopicName} select * from $BucketName:${bucketSetup.PrefixName}/$dir/ STOREAS `${format.entryName}$formatExtensionString` LIMIT 190",
+                KCQL_CONFIG -> s"insert into ${bucketSetup.TopicName} select * from $BucketName:${bucketSetup.PrefixName}/$dir/ STOREAS `${format.entryName}$formatExtensionString` LIMIT 190",
+              ),
+            ).asJava
+
+          task.start(props)
+          //Let the partitions scan do its work
+          Thread.sleep(4000)
```

Comment:
How would you feel about using `eventually` instead of adding artificial delays to the tests, which can introduce unreliability and slow the tests down unnecessarily?

I added a similar one to this yesterday, the example is here:
https://github.com/lensesio/stream-reactor/blob/cf3fca08f83043a59d2e73aa10b9e8c01451fba3/kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/source/S3SourceTaskXmlReaderTest.scala#L62

Reply:
Eventually is good, but we know it will fail once or twice. I will adapt the tests. For me it does not give more in this case.